spec/fixtures/integration/ielr.y

```diff
@@ -0,0 +1,62 @@
+%{
+#include <stdio.h>
+#include <stdlib.h>
+#include "y.tab.h"
+#define YYDEBUG 1
+static int yylex(YYSTYPE *val, YYLTYPE *loc);
+static int yyerror(YYLTYPE *loc, const char *str);
+%}
+
+%union {
+    int val;
+}
+
+%token a
+%token b
+%token c
+%define lr.type ielr
+
+%%
```

Comment:
This grammar example comes from Fig 5 of the original paper. Could clarify it as comments?